crypto/x509/x_pubkey.c

```diff
@@ -219,8 +219,10 @@ EVP_PKEY *X509_PUBKEY_get0(X509_PUBKEY *key)
 EVP_PKEY *X509_PUBKEY_get(X509_PUBKEY *key)
 {
     EVP_PKEY *ret = X509_PUBKEY_get0(key);
```

Comment:
Please add a blank line after this declaration

Reply:
ok.